test/io/test_storage.py

```diff
@@ -91,7 +90,7 @@ def s3_storage(aws_credentials: Any):
         ("s3_storage", "sample_dataset"),
     ],
 )
-def test_storage_saves_and_fetches_data(
+def test_storage_saves_data(
```

Comment:
Shouldn't need any changes in this file, except maybe to add a test to verify that you can change the storage path using some new parameters. But no existing tests should need to be changed